CHANGELOG.md

```diff
@@ -74,6 +74,7 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Added `ownMessageText` and `otherMessageText` to `StreamColors` to enable message text customization. [#4175](https://github.com/GetStream/stream-chat-android/pull/4175)
```

Comment:
I'd just expand this a bit more - I'd add: "If you've been using `StreamColors.textHighEmphasis` to customize the color of message texts, we recommend using the new attributes instead.